docs/backup-restore.md

```diff
@@ -1,32 +1,96 @@
+If you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL,
+you can restore to any point within the backup retention period with the automated backup feature.
+
+### Cassandra
+
+You first need to stop all the nodes of a Cassandra cluster. Clean the directories (`data`, `commitlogs`, and `hints`) and place backups (snapshots) in each node. Then, start all the nodes.
+
+To avoid mistakes, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+### Cosmos DB
+
+You can follow the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and change the default consistency to `STRONG` after restoring the backup.
+It is recommended to use the mid-time of paused duration as a restore point as we explained earlier.
+
+### DynamoDB
+
+You can basically follow [the official doc](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.Tutorial.html). However, a table can only be restored with an alias, so you need to restore a table with an alias and delete the original table and rename the alias to the original name to restore the same named tables.
+
+Specifically, please follow the steps below.
+
+* Select the mid-time of paused duration as the restore point.
+* Restore with PITR of table A to another table B
+* Take a backup of the restored table B (assume the backup is named backup B)
+* Remove table A
+* Create a table named A with backup B
 
-## Scalar DB server
+Not that tables can only be restored one by one so you need to do the above steps for each table.
 
-Since Scalar DB server implements [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, you can easily pause the server to stop accepting incoming requests and drain in-flight requests.
-You can use paused duration and choose one of the following ways depending on the underlining database to make your databases transactionally-consistent even after restored.
-* Take backups of your databases during the paused duration with snapshot feature.
-* Restore to a point in the paused duration with point-in-time-restore (PITR) feature.
+Note also that some of the configurations such as PITR and auto scaling policies are reset to the default values for restored tables, so please manually configure required settings. See [the doc](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CreateBackup.html#CreateBackup_HowItWorks-restore) for more detail.
+If you used the schema loader and didn't change any other configurations, executing the schema loader with the same schema file is one of the ways to restore the original configuration. It only sets missing configurations if a specified table exists.
```

Comment:
@brfrn169 Thank you! 

@scalar-boney Please check. If it throws an exception, I will update the doc.

Reply:
@feeblefakie san, @scalar-boney 
I have checked with schema loader as per the comment, my findings are added below
When re applying schema using scalar db schema loader(ex .`scalardb-schema-loader-3.4.0.jar`), if a table already exists a warning is shown that table already exists(ex .`2021-12-21 18:24:36,399 [WARN  scalar-schema.dynamo] qa.question already exists`).
But in case of a restored table, features like PITR and auto-scaling are not enabled back when re applying schema with schema loader(ex .`scalardb-schema-loader-3.4.0.jar`).
With earlier versions of schema loader( ex. `scalar-schema-standalone-3.2.2.jar`), when trying to re apply schema, features like PITR and auto-scaling are enabled for restored table.
In both schema loader(ex .`scalardb-schema-loader-3.4.0.jar`) and standalone tool( ex. `scalar-schema-standalone-3.2.2.jar`), when re applying schema if tables already exist, only warnings are shown that the table exists, no error has occurred.
We may need to update and change regarding PITR being enabled on re applying schema to a restored table using schema loader since newer schema loader versions are not enabling that if a table already exists.